charts/grafana/values.yaml

```
@@ -435,10 +436,10 @@ enableServiceLinks: true
 
 ## Additional grafana server secret mounts
 # Defines additional mounts with secrets. Secrets must be manually created in the namespace.
-extraSecretMounts: []
-  # - name: secret-files
-  #   mountPath: /etc/secrets
-  #   secretName: grafana-secret-files
+extraSecretMounts: 
```

I think it's better to set empty array by default to not bring possible issues 
```suggestion
extraSecretMounts: []
```